src/clinical/clinical-service.ts

```diff
@@ -122,6 +124,10 @@ export async function getDonors(programId: string) {
   return await donorDao.findBy({}, 999);
 }
 
+export async function getDonorsByIds(donorIds: number[]) {
+  return await donorDao.findByDonorIds(donorIds);
```

Comment:
is this `await` necessary here? the function being async means the return will be a promise regardless, and the result can (or should?) be awaited there.

I recall seeing this be pointed out by ESLint at some point, but maybe I'm confusing things.

Reply:
nice catch. not sure if makes a difference but yes this await isn't needed